docs/quickstart.md

```diff
@@ -32,47 +32,28 @@ These instructions help you install the CLI.
 
 ### Install `conda` and `conda-forge`
 
-The easiest way to install MyST is with [the `conda` package manager](https://conda.io/projects/conda/en/latest/).
-`conda` is a multi-language manager that is useful for users across many data science languages like Python, R, Julia, and JavaScript.
+The easiest way to install MyST is with [the `mamba`/`conda` package manager](https://mamba.readthedocs.io/en/latest/index.html).
+`mamba` is a multi-language manager that is useful for users across many data science languages like Python, R, Julia, and JavaScript.
 
-The easiest way to install `conda` is with [the `conda-forge` distribution](https://conda-forge.org/download/).
-This includes miniconda and a community-driven package index called `conda-forge`.
+The easiest way to install `mamba` is with [the `miniforge` distribution](https://github.com/conda-forge/miniforge?tab=readme-ov-file).
 
-🛠 Install miniconda from [conda-forge](https://conda-forge.org):
+🛠 Install [miniforge](https://github.com/conda-forge/miniforge):
 
-1. Go to [the conda-forge latest releases section](https://conda-forge.org/download/).
+1. Go to [the `miniforge` Downloads section](https://github.com/conda-forge/miniforge?tab=readme-ov-file#download).
 2. Download the release for your platform and follow the instructions.
 3. To check that you've installed properly, you should be able to execute the following command to print the version of conda:
 
    ```shell
-   $ conda --version
-   conda 24.7.1
+   $ mamba --version
+   mamba 1.5.8
+   conda 24.7.1   
    ```
 
```

Comment:
````suggestion
   mamba 1.5.8
   conda 24.7.1   
   ```

:::{seealso} There are many ways to install MyST
This tutorial focuses on the simplest possible way to install MyST from scratch.
If you're familiar with either Python or JavaScript-based workflows, see [](installing.md) for more information on how to install MyST.
:::
````